app/kubernetes/converters/application.js

```diff
@@ -169,7 +171,7 @@ class KubernetesApplicationConverter {
       data.spec.template.spec.volumes,
       (acc, volume) => {
         if (volume.configMap || volume.secret) {
-          const matchingVolumeMount = _.find(data.spec.template.spec.containers[0].volumeMounts, { name: volume.name });
+          const matchingVolumeMount = _.find(_.flatMap(_.map(containers, 'volumeMounts')), { name: volume.name });
```

Comment:
If I'm right you can use `_.find(_.flatMap(containers, 'volumeMounts'), { name: volume.name });`